source/features/extend-conversation-status-filters.tsx

```diff
@@ -1,80 +1,67 @@
-import React from 'dom-chef';
-import {$, $$} from 'select-dom';
-import CheckIcon from 'octicons-plain-react/Check';
-import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 import SearchQuery from '../github-helpers/search-query.js';
+import observe from '../helpers/selector-observer.js';
 
-function addMergeLink(): void {
+function getFilterState(): {isMerged: boolean; isUnmerged: boolean} {
```

Comment:
This can be inlined in `addMergeLink`, like:

```js
const locationQuery = SearchQuery.from(location);
const isMerged = locationQuery.includes('is:merged');
const isUnmerged = locationQuery.includes('is:unmerged');
```